pyproject.toml

```diff
@@ -85,7 +84,7 @@ write_to = "src/_pytest/_version.py"
 
 [tool.black]
 target-version = [
-    'py38',
+    'py39',
```

Comment:
Side note, according to https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#t-target-version, we don't need to configure this because black also looks at `requires-python`. It also says we're actually supposed to list all supported versions, oh well.